bin/sl-deploy.js

```diff
@@ -12,11 +13,13 @@ var fs = require('fs');
 var maybeTunnel = require('strong-tunnel');
 var path = require('path');
 
-function printHelp($0, prn) {
-  var USAGE = fs.readFileSync(require.resolve('./sl-deploy.txt'), 'utf-8')
-      .replace(/%MAIN%/g, $0)
-      .trim();
+if (path.basename(process.cwd()) === 'test')
```

Comment:
That way, the following commands work for strong-deploy.  But, still don't feel good about other modules...still an open issue.

 cd strong-deploy
 npm test
 node test/test-*
 cd test
 node test-*